qt-bugs@ issue : 197621
Trolltech task ID : 197836
bugs.kde.org number : none
applied: no
author: Montel Laurent <montel@kde.org>

This patch fixes crashs in q3stylesheet (it was possible to
use a qfont size < 1)
Bug reported to TT with this patch. 

--- a/src/qt3support/text/q3stylesheet.cpp
+++ b/src/qt3support/text/q3stylesheet.cpp
@@ -1463,9 +1463,9 @@
         s = baseSize;
     }
     if (pixel)
-        font.setPixelSize(s);
+        font.setPixelSize(qMax(1,s));
     else
-        font.setPointSize(s);
+        font.setPointSize(qMax(1,s));
 }
 
 QT_END_NAMESPACE
